index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
+    'consistent-return': 'off',
+    'arrow-body-style': ['warn', 'as-needed', {
+      requireReturnForObjectLiteral: false,
+    }],
+    'prefer-rest-params': 'warn',
+    'prefer-destructuring': ['warn', {
+      VariableDeclarator: {
+        array: false,
+        object: true,
+      },
+      AssignmentExpression: {
+        array: true,
+        object: true,
+      },
+    }, {
+      enforceForRenamedProperties: false,
+    }],
+
+    'no-underscore-dangle': 'off',
+    'array-element-newline': 'off',
+    'function-paren-newline': ['warn', 'multiline'],
+    'newline-per-chained-call': ['warn', { ignoreChainWithDepth: 4 }],
```

Comment:
I'm not in favour of changing this to `warn`. I think there are great readability improvements with having newlines per chained call. Plus, as pointed out in https://eslint.org/docs/rules/newline-per-chained-call it helps see what's actually changed in diffs. I also think a chain of 4 is too high of a threshold.

Reply:
4 is the default (https://github.com/airbnb/javascript/blob/7ff6303513b46e8b7dd0bc03327b138a6db7b3b4/packages/eslint-config-airbnb-base/rules/style.js#L234) , yet this (https://github.com/CoderDojo/coolest-platform/commit/f4c4d09c074252a243b025eef0dc63e4923e9d00#diff-9e4a215b4f5f613a802fa43d8ab35270R67) has been reworked.
Considering what the notion of "chain" is (like (https://lodash.com/docs/4.17.4#lodash) ) a chaining doesn't necessarly involve an "action".
Grouping by "role" in a chain makes way more sense to me.

Reply:
Personally, I just find one per line so much more readable.

We can deal with this on a case-by-case basis in code review I guess.